Source/JavaScriptCore/wasm/WasmTable.cpp

```diff
@@ -88,9 +89,14 @@ RefPtr<Table> Table::tryCreate(uint32_t initial, std::optional<uint32_t> maximum
     switch (type) {
     case TableElementType::Externref:
         return adoptRef(new ExternRefTable(initial, maximum));
-    case TableElementType::Funcref:
+    case TableElementType::Funcref: {
+        if (maximum && maximum.value() == initial) {
```

Comment:
Why not check `if (isFixedSized())` instead?

Reply:
This is static function, to create a table, so isFixedSized is not computed (it is computed after creating a table).

Reply:
Oh right.  I mistook this for being part of the constructor above.